core/pom.xml

```diff
@@ -115,11 +115,6 @@
             <groupId>org.apache.spark</groupId>
             <artifactId>spark-sql_${scala.version.prefix}</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.apache.hadoop</groupId>
-            <artifactId>hadoop-common</artifactId>
-            <scope>test</scope>
```

Comment:
Thanks for catching and fixing the duplicate dependency. However, the two dependencies have different scopes. The tests still pass if the other one is removed. However, I am uncertain if the bundled jar (cli) would function, since hadoop-common has a provided scope in the parent. What do you think is the appropriate scope?

Reply:
I don't think the scope here should be a test, just use the default scope